src/Tasks.UnitTests/CreateItem_Tests.cs

```diff
@@ -154,7 +154,7 @@ public void WildcardsWithRecursiveDir()
             ObjectModelHelpers.DeleteTempProjectDirectory();
 
             ObjectModelHelpers.CreateFileInTempProjectDirectory("Myapp.proj", @"
-                <Project ToolsVersion=`msbuilddefaulttoolsversion` xmlns=`msbuildnamespace`>
+                <Project ToolsVersionresol=`msbuilddefaulttoolsversion` xmlns=`msbuildnamespace`>
```

Comment:
?

Reply:
Not sure how that crept in; will revert.